package.json

```diff
@@ -1,131 +1,19 @@
-    "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each"
-  },
-  "dependencies": {
-    "broccoli-funnel": "^3.0.8",
-    "broccoli-merge-trees": "^3.0.2",
-    "common-tags": "^1.8.0",
-    "ember-auto-import": "^2.6.1",
-    "ember-cli-babel": "^7.26.11",
-    "ember-cli-test-loader": "^3.0.0",
-    "resolve-package-path": "^4.0.3",
-    "silent-error": "^1.1.1",
-    "validate-peer-dependencies": "^2.2.0"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.21.0",
-    "@babel/eslint-parser": "^7.19.1",
-    "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.9.3",
-    "@embroider/test-setup": "^2.1.1",
-    "@glimmer/component": "^1.1.2",
-    "@tsconfig/ember": "^2.0.0",
-    "@types/qunit": "^2.19.4",
-    "@types/rsvp": "^4.0.4",
-    "ember-angle-bracket-invocation-polyfill": "^3.0.2",
-    "ember-cli": "~4.10.0",
-    "ember-cli-dependency-checker": "^3.3.1",
-    "ember-cli-htmlbars": "^6.2.0",
-    "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-terser": "^4.0.2",
-    "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-load-initializers": "^2.1.2",
-    "ember-resolver": "^9.0.1",
-    "ember-source": "~4.10.0",
-    "ember-source-channel-url": "^3.0.0",
-    "ember-try": "^2.0.0",
-    "eslint": "^8.35.0",
-    "eslint-config-prettier": "^8.6.0",
-    "eslint-plugin-disable-features": "^0.1.3",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "expect-type": "^0.15.0",
-    "loader.js": "^4.7.0",
-    "npm-run-all": "^4.1.5",
-    "prettier": "2.8.4",
-    "qunit": "^2.19.4",
-    "release-it": "^15.7.0",
-    "release-it-lerna-changelog": "^5.0.0",
-    "typescript": "^4.9.5",
-    "webpack": "^5.75.0"
-  },
-  "peerDependencies": {
-    "@ember/test-helpers": "^2.9.3",
-    "ember-source": ">=3.28",
-    "qunit": "^2.13.0"
-  },
-  "engines": {
-    "node": "14.* || 16.* || >= 18"
-  },
-  "publishConfig": {
-    "registry": "https://registry.npmjs.org"
-  },
-  "changelog": {
-    "repo": "emberjs/ember-qunit",
-    "labels": {
-      "breaking": ":boom: Breaking Change",
-      "enhancement": ":rocket: Enhancement",
-      "bug": ":bug: Bug Fix",
-      "documentation": ":memo: Documentation",
-      "internal": ":house: Internal"
-    }
-  },
-  "ember": {
-    "edition": "octane"
-  },
-  "ember-addon": {
-    "configPath": "tests/dummy/config"
-  },
-  "release-it": {
-    "plugins": {
-      "release-it-lerna-changelog": {
-        "infile": "CHANGELOG.md",
-        "launchEditor": true
-      }
-    },
-    "git": {
-      "tagName": "v${version}"
-    },
-    "github": {
-      "release": true,
-      "tokenRef": "GITHUB_AUTH"
-    }
+    "lint": "yarn workspaces run lint",
+    "lint:fix": "yarn workspaces run lint:fix",
+    "test": "yarn workspaces run test"
   },
+  "workspaces": [
+    "addon"
+  ],
   "volta": {
-    "node": "14.19.1",
-    "yarn": "1.22.18"
+    "node": "16.20.0",
```

Comment:
According to `addon/package.json` node 14 is still supported.

I guess dropping that support in a separate PR would be better for changelog entries.

Reply:
exactly, yeah, and I don't want to have PRs have more purposes than stated.
Once folks start merging these PRs, it'll be much easier (for review, and the changelog) to go through the details like this.

Node 14 already has to be dropped because `@ember/test-helpers` dropped it